android/src/com/mapswithme/maps/routing/NavigationController.java

```diff
@@ -59,7 +59,7 @@
 
   private final SearchWheel mSearchWheel;
 
-  private boolean mShowTimeLeft = true;
+  private static boolean mShowTimeLeft = true;
```

Comment:
Hi @alexzatsepin , I totally  agree with you. What do you think about my last change ?